### PR TITLE
Add new metrics to API

### PR DESCRIPTION
**Pull request by apozdeyev**
## Changes

Add new metrics to API: network_growth_change_1d/7d/30d

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json [hunk 1]
```diff
@@ -2236,5 +2236,56 @@
     "table": "intraday_metrics",
     "has_incomplete_data": false,
     "data_type": "timeseries"
+  },
+  {
+    "human_readable_name": "Network Growth Change (1d)",
+    "name": "network_growth_change_1d",
+    "metric": "network_growth_change_1d",
+    "version": "2019-01-01",
+    "access": "free",
```
Comment (tspenov) [inline lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:2245]: We need to add access `restricted`. That means that `free` user can't have realtime and all historical data.

Comment (apozdeyev) [reply lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:2245]: ok, but what about `min_plan`, leave it as is?

Comment (tspenov) [reply lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:2245]: @apozdeyev yes, `min_plan` means that only this plan or higher can access this metric and lower plans can't even see the restricted data.

Comment (apozdeyev) [reply lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json]: Done!

**Pull request merged**